PlainTasks.py

```diff
@@ -28,6 +28,14 @@
     import subprocess
 
 
+def tznow():
+    return datetime.now().replace(
```

Comment:
If the timezone is passed to datetime.datetime.new() and timezone info is supported for the platform (gettimeofday) resulting datetime object will a conversion from local timezone to the specified one. In contrast, datetime.datetime.replace() just changes the appropriate property.

https://docs.python.org/3.5/library/datetime.html#datetime.datetime.now